src/couch/src/couch_httpd_auth.erl

```diff
@@ -19,8 +19,10 @@
 
 -export([party_mode_handler/1]).
 
--export([default_authentication_handler/1, default_authentication_handler/2,
-	 special_test_authentication_handler/1]).
+-export([
+    default_authentication_handler/1, default_authentication_handler/2,
```

Comment:
I wish it erlfmt have control where to keep the original formatting and where it should be rewriten. In thins case we might want to manually split exports one per line.